README.md

```diff
@@ -84,6 +85,15 @@ Note that, using context exludes `{ block: 'button'}` from result.
 
 So `parse('m:theme=normal', { block: 'button' })` is not same as `parse('b:button m:theme=normal')`
 
+### stringify
+
+Parameter | Type     | Description
+----------|----------|------------------------------------------------------------------------------
+`entities`| `array`  | Array of [BEM entities] to merge into bem-import string [notation](#notation)
+
+Forms a string from [bem entities]. Be aware to merge only one type of entities.
+Array must contain block and it's mods, or elem and it's mods.
```

Comment:
modifiers

The array should contains one block or one elem and optionally it's modifiers.